content/master/getting-started/introduction.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Crossplane Introduction
+weight: 2
+---
+
+Crossplane connects your Kubernetes cluster to external,
+non-Kubernetes resources, and allows platform teams to build custom Kubernetes
+APIs to consume those resources.
+
+Crossplane creates Kubernetes
+[Custom Resource Definitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
+(`CRDs`) to represent the external resources as native 
+[Kubernetes
+objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/). 
+As native Kubernetes objects, you can use standard commands like `kubectl create`
+and `kubectl describe`. The full 
+[Kubernetes API](https://kubernetes.io/docs/reference/using-api/) is available
+for every Crossplane resource. 
+
+Crossplane also acts as a
+[Kubernetes Controller](https://kubernetes.io/docs/concepts/architecture/controller/)
+to monitor the state of the external resources and provide state enforcement. If
+something modifies or deletes a resource outside of Kubernetes, Crossplane reverses
+the change or recreates the deleted resource.
```

Comment:
```suggestion
something modifies or deletes a resource outside of Kubernetes, Crossplane attempts to reverse
the change or recreates the deleted resource.
```

small wording suggestion, as it will respect the external API if the reversal is not allowed

Reply:
I'm okay with this suggestion. One thing I'm attempting to do here is provide a minimal amount of information and relying on later sections to explain details and caveats. 

Do you feel this is critical to someone brand new to Crossplane or can we put this in a later section describing the Controller behavior in more detail?

Reply:
Not critical in my opinion, and I agree with your approach for intro material (was going for a minimal diff). I'm good with your alternative as well.